examples/docker-compose.example.yml

```diff
@@ -41,7 +41,7 @@ services:
     volumes:
       - "/path/to/library:/romm/library"
       - "/path/to/resources:/romm/resources" # [Optional] Path where roms metadata (covers) are stored
-      - "/path/to/config.yml:/romm/config.yml" # [Optional] Path where config is stored
+      - "/path/to/config:/romm/config" # [Optional] Path where config is stored
```

Comment:
we'll need to mark this as a breaking change in 3.0 release

Reply:
Yes absolutely!